test/v2/init.js

```diff
@@ -24,38 +24,46 @@ var test = require('tape');
 var testRead = require('../lib/read_test.js');
 var Init = require('../../v2/init.js');
 
+/* jshint camelcase:false */
+
 var testInitReq = Buffer([
     0x00, 0x02,             // version:2
-
+    0x00, 0x03,             // nh:2
```

Comment:
nope, the comment is to remind that it's a 2-byte field, it's just a line-wise decomposition of the comment banner above each class in e.g. v2/{init,call,frame}